core/domain/user_services.py

```diff
@@ -2815,6 +2749,11 @@ def log_username_change(committer_id, old_username, new_username):
         new_username=new_username).put()
 
 
+def get_auth_claims_from_request(request):
+    """Returns AuthClaims associated to the user who authorized the request."""
+    return auth_services.authenticate_request(request)
```

Comment:
authenticate_*() doesn't sound like it'll return anything. Can you have a name that implies "get"? Maybe literally get_auth_claims_from_request()?

Also don't forget to add args, returns, etc.

Reply:
Done.